eng/pipelines/templates/jobs/vmr-build.yml

```diff
@@ -460,18 +460,43 @@ jobs:
 
   # Only upload test results if enabled
   - ${{ if eq(parameters.runTests, 'True') }}:
+    - ${{ if eq(parameters.buildSourceOnly, 'True') }}:
+      - task: PublishTestResults@2
+        displayName: Publish Source Build Smoke Test Results
+        condition: succeededOrFailed()
+        continueOnError: true
+        inputs:
+          testRunner: VSTest
+          testResultsFiles: 'test/**/*.trx'
+          searchFolder: $(sourcesPath)
+          mergeTestResults: true
+          publishRunAttachments: true
+          testRunTitle: SourceBuild_SmokeTests_$(Agent.JobName)
+
```

Comment:
This block should be removed. Smoke tests don't publish test results into the test folder anymore.

Reply:
Fixed in https://github.com/dotnet/installer/pull/19222/commits/8e9b075f98eb1bb5ec618d1964b8bf679c271d71